### DISCUSSION TITLE
Microsoft Edge switching from React UI to WebUI 2

### DISCUSSION THREAD

**Discussion by enimiste** [webui-dev/webui#386] [General]
Hi everyone,

I just saw a new announcement from Microsoft about migrating Microsoft Edge from `React UI` to `WebUI 2`. This is the first time I have heard about WebUI, so I did some research which led me to this library and repository.

**My question is: is this the library that Microsoft uses for Edge, or are they referring to a different one?**

Thanks

**Comment by hassandraga** [comment]
Microsoft WebUI 2.0 is a new markup-first UI architecture. It's an internal Microsoft project designed to improve Edge's UI performance. Our open-source WebUI 2.0 library uses any installed web browser as GUI. Just a similar name.

_Source: https://blogs.windows.com/msedgedev/2024/05/28/an-even-faster-microsoft-edge/_
